ckanext/recombinant/templates/datatables/datatables_view.html

```diff
@@ -0,0 +1,70 @@
+{% ckan_extends %}
+
+{%- block datatable_field_headers -%}
+
+  {% set chromo = h.recombinant_published_resource_chromo(resource.id) %}
+  {% if chromo %}
+
+    {% set chromo_keyed = {} %}
+    {% for recombinant_field in chromo.fields %}
+      {% set _ = chromo_keyed.update({recombinant_field.datastore_id: {'description': recombinant_field.description,
+                                                                       'label': recombinant_field.label}}) %}
+    {% endfor %}
+
+    {%- set datadictionary_fields = h.datastore_dictionary(resource.id) -%}
+
+    {% for field in datadictionary_fields %}
+
+      {% if 'show_fields' not in resource_view or field.id in resource_view.show_fields -%}
+
+        {% if field.id in chromo_keyed %}
+        {# the DS field is a Recombinant field, use chromo #}
+          {% set description = h.recombinant_language_text(chromo_keyed[field.id].description) | replace('\n', '<br>' | safe) %}
+          {% set label = h.recombinant_language_text(chromo_keyed[field.id].label) | trim %}
+          <th scope="col">
+            {{ label }}&nbsp;
+            {% if description|length %}
```

Comment:
empty strings are falsy
```suggestion
            {% if description %}
```